user/example/misc/scan/ExampleScan.scan

```diff
@@ -0,0 +1,27 @@
+[global]
+repeatScans = 1
+allowNested = 0 # overrides local nested arguments
+configPrefix = "/home/lhuth/test/scanConf/scanned"
```

Comment:
Is it possible be  a relateive path?

Reply:
It's up to the user to set the path to be relative - this makes it, however, execution folder dependent and I do not recommend to do so.